XU-TechnicalPaper.md

```diff
@@ -127,29 +137,19 @@ multicast. New XU node coming online: ‘sync old orderbook for subscribed chann
 
-3. XU members on UnicornCoin update their orderbooks based on this new order.
+3.6.3. XU members on UnicornCoin update their orderbooks based on this new order.
 
-4. If another XU member, `Exchange B`, wants to fill this 100 UnicornCoin order, the XU node takes care of the cross-chain atomic swap which means `Exchange A` gets **100 UnicornCoin** and `Exchange B` gets **10 BTC** on their payment channels on the respective chains. Behind the scenes, both parties create transactions on both chains to setup a trade which imposes time constraints i.e. if a trade is not successful both parties get their money back. Order filling is realized on a first-come-first-served basis.
+3.6.4. If another XU member, `Exchange B`, wants to fill this 100 UnicornCoin order, the XU node takes care of the cross-chain atomic swap which means `Exchange A` gets **100 UnicornCoin** and `Exchange B` gets **10 BTC** on their payment channels on the respective chains. Behind the scenes, both parties create transactions on both chains to setup a trade which imposes time constraints i.e. if a trade is not successful both parties get their money back. Order filling is realized on a first-come-first-served basis.
 
-```
-[TODO]
 Exceptions:
-An exchange can choose to subscribe to the orderbook of particular exchanges only (e.g. through legal requirements)
-
-- Order is filled by another user who was faster than me: funds WITH error message come back and XU 
-node starts looking for the next best order. We should use the ‘odd type’ messaging system as 
-specified in the RFC. E.g. Type 17 for errors.
-
-- Exchange B goes offline: time-out message comes back and XU node starts looking for the next best 
-order. Exchange B repeatedly doesn’t fill orders: punishment, see chapter 3.4. 
-
-- Trade volume for a particular asset exceeds payment channel volume. XU node maintains a configurable 
-threshold and automatically redeposits from an authorized hot wallet. Details, sources needed.
-```
+- An exchange can choose to subscribe to the orderbook of particular exchanges only (e.g. through legal requirements)
+- Order is filled by another user who was faster than me: funds WITH error message come back and XU node starts looking for the next best order. We should use the ‘odd type’ messaging system as specified in the RFC. E.g. Type 17 for errors.
+- Exchange B goes offline: time-out message comes back and XU node starts looking for the next best order. Exchange B repeatedly doesn’t fill orders: punishment, see chapter 3.4. 
+- Trade volume for a particular asset exceeds (own) payment channel volume. XU node maintains a configurable threshold and automatically redeposits from an authorized hot wallet. Details, sources needed.
```

Comment:
Trade volume for a particular asset exceeds (own) payment channel volume: **(Missing semicolon)**